src/main/kotlin/no/nav/familie/ef/sak/service/PersonopplysningerService.kt

```diff
@@ -58,7 +60,7 @@ class PersonopplysningerService(private val personService: PersonService,
                 telefonnummer = søker.telefonnummer.find { it.prioritet == 1 }
                         ?.let { TelefonnummerDto(it.landskode, it.nummer) },
                 statsborgerskap = søker.statsborgerskap.map {
-                    StatsborgerskapDto(land = it.land,
+                    StatsborgerskapDto(land = adresseMapper.hentLand(it.land, datoEllerIdag(it.gyldigFraOgMed)),
```

Comment:
private fun lagPersonopplysninger burde kanskje ligget it PersonopplysnningerMapper og hete toDto ?

Reply:
Fikset i d6219bb